### ISSUE TITLE
Available table status.

### ISSUE THREAD

**Issue by ilyakava** [ilyakava/pongbot#2]
Allow configuring number of tables, and asking about ongoing matches to see if a table is available.

**Comment by jinpark** [comment]
So, the game plan is, if 1 table, and if there is an accepted challenge, mark table as used?

**Comment by ilyakava** [comment]
yep, just compare an env var of tables_available to the number of current ongoing challenges
